### ISSUE TITLE
BB pod - `upload` raises exception: Message:  No reader function for tag object

### ISSUE THREAD

**Issue by sheluchin** [epiccastle/spire#151]
I'm getting the following exception when trying to `upload` something:

```
$ grep spire bb.edn
 :pods {epiccastle/spire {:version "0.1.0-alpha.18"}}
$ bb --version
babashka v0.8.1
src/tasks/server.clj:63 (upload/upload {:src "/home/alex/repos/example/.secrets/id_rsa", :dest "/home/ubuntu/.ssh/", :preserve false, :mode 384})
#error {
 :cause No reader function for tag object
 :via
 [{:type java.lang.RuntimeException
   :message No reader function for tag object
   :at [clojure.lang.EdnReader$TaggedReader readTagged EdnReader.java 801]}]
 :trace
 [[clojure.lang.EdnReader$TaggedReader readTagged EdnReader.java 801]
  [clojure.lang.EdnReader$TaggedReader invoke EdnReader.java 783]
  [clojure.lang.EdnReader$DispatchReader invoke EdnReader.java 549]
  [clojure.lang.EdnReader readDelimitedList EdnReader.java 757]
  [clojure.lang.EdnReader$MapReader invoke EdnReader.java 680]
  [clojure.lang.EdnReader readDelimitedList EdnReader.java 757]
  [clojure.lang.EdnReader$MapReader invoke EdnReader.java 680]
  [clojure.lang.EdnReader readDelimitedList EdnReader.java 757]
  [clojure.lang.EdnReader$ListReader invoke EdnReader.java 654]
  [clojure.lang.EdnReader read EdnReader.java 145]
  [clojure.lang.EdnReader read EdnReader.java 111]
  [clojure.lang.EdnReader readString EdnReader.java 67]
  [clojure.edn$read_string invokeStatic edn.clj 46]
  [clojure.edn$read_string invokeStatic edn.clj 37]
  [spire.pod.core$main$fn__28230 invoke core.clj 1208]
  [spire.pod.core$main invokeStatic core.clj 1208]
  [spire.core$_main invokeStatic core.clj 67]
  [spire.core$_main doInvoke core.clj 59]
  [clojure.lang.RestFn invoke RestFn.java 397]
  [clojure.lang.AFn applyToHelper AFn.java 152]
  [clojure.lang.RestFn applyTo RestFn.java 132]
src/tasks/server.clj:63 (upload/upload {:src "/home/alex/repos/example/.secrets/id_rsa", :dest "/home/ubuntu/.ssh/", :preserve false, :mode 384}) ubuntu@localhost: 2222
----- Error --------------------------------------------------------------------
Type:     clojure.lang.ExceptionInfo
```

The operation actually succeeds - the file is there - but the execution is interrupted.

**Comment by retrogradeorbit** [comment]
This is actually caused by the progess bar. All my testing on bb has been with the status updates turned off. A fix is in the works.

**Comment by retrogradeorbit** [comment]
fixed in 0.1.1

**Comment by sheluchin** [comment]
Thank you, @retrogradeorbit!
